backend/imageupload/migrations/0002_rename_imageuploads_imageupload.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 4.1.5 on 2023-03-05 04:45
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("imageupload", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.RenameModel(
+            old_name="ImageUploads",
+            new_name="ImageUpload",
```

Comment:
Just renaming this model, the convention in Django is the name model using singular noun